components/Common/AvatarGroup/Avatar/index.tsx

```diff
@@ -0,0 +1,29 @@
+import type { FC } from 'react';
+import * as RadixAvatar from '@radix-ui/react-avatar';
+import styles from './index.module.css';
+
+type AvatarProps = {
+  src?: string;
+  alt?: string;
+  extraClass?: string;
+  onClick?: () => void;
+};
+
+const usrnameToFallbackText = (name?: string) => {
```

Comment:
You should assume an alt is provided or simply do `name: string = ''`

Reply:
gotcha !